defaults/main.yml

```diff
@@ -552,7 +548,7 @@ maas_pip_packages:
   - python-keystoneclient
   - python-magnumclient
   - python-neutronclient
-  - python-novaclient<7.0.0
+  - python-novaclient
```

Comment:
well, ocata is 'unofficially' capped at <7.2.0 or the very least <8.0.0 (which we know is pike)